CrewChiefV4/Events/Fuel.cs

```diff
@@ -426,6 +429,46 @@ private Boolean reportFuelConsumption()
             return haveData;
         }
 
+        private Boolean reportFuelConsumptionForLaps(int numberOfLaps)
+        {
+            Boolean haveData = false;
+            if (fuelUseActive && usagePerLap.Count > 0)
+            {
+                // round to 1dp
+                float meanUsePerLap = (((float)Math.Round(usagePerLap.Average() * 10f)) / 10f ) * numberOfLaps;
```

Comment:
i think the rounding hack can be moved here to make the rounded value more accurate (round if after multiplying, not before)

float meanUsePerLap = ((float)Math.Round(usagePerLap.Average() * numberOfLaps * 10f)) / 10f;

Reply:
ok will push that change aswell